.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# https://help.github.com/en/articles/about-code-owners#codeowners-syntax
```

Comment:
Rebase done. Thank you

Reply:
Heads up, I am still seeing these files included in the pull request. Perhaps your remote wasn't pointed to the right repository when you did the rebase so you rebased against _your_ repository instead of this one?